cvmfs/loader.h

```diff
@@ -156,6 +156,7 @@ struct LoaderExports {
 
   // added with CernVM-FS 2.1.8 (LoaderExports Version: 2)
   bool disable_watchdog;
+  bool simple_options_parsing;
```

Comment:
Please add the necessary versioning information according to the documentation of this struct.